### DISCUSSION TITLE
Company Logo added to the <nav class> for The Blog Theme

### DISCUSSION THREAD

**Discussion by TimTracy** [OrchardCMS/OrchardCore#9562] [Q&A]
The easiest way to add a Media file from the Media Library to appear in the  navigation area <nav class> for The Blog Theme.
 Would like to do from Admin dashboard with template or layout.
Presently Site Name is displayed. Would like to add a logo image to appear after the Site Name.
Will be doing this for several Web Sites and do not want to modify The Blog Theme in visual studio.

Thanks.

**Comment by PiemP** [comment]
https://docs.orchardcore.net/en/dev/docs/reference/modules/CustomSettings/

You can setup the custom settings from setup recipe.

**Reply by TimTracy** [reply]
I have setup a Company Brand with Name(text) and Logo(Media) and selected the logo from the media library.
Not sure where I can reference the logo to appear in the Navigation bar.

**Reply by Piedone** [reply]
With The Blog Theme you'll need to override the Layout template, what you can also do from the dashboard with Templates. See the Layout template: https://github.com/OrchardCMS/OrchardCore/blob/dev/src/OrchardCore.Themes/TheBlogTheme/Views/Layout.liquid In there you can then access your custom settings (read the docs about it: https://docs.orchardcore.net/en/dev/docs/reference/modules/Liquid/).
